emukit/core/acquisition/integrated_acquisition.py

```diff
@@ -0,0 +1,63 @@
+from typing import Union, Callable, Tuple
+
+import numpy as np
+
+from emukit.core.acquisition import Acquisition
+from emukit.core.interfaces import IModel, IPriorHyperparameters, IDifferentiable
+
+
+class IntegratedAcquisition(Acquisition):
```

Comment:
This might be confused with integrating over the model rather than its hyperparameters. For example "global variance reduction" acquisition does not fit in this scheme. I don't have a very good name though, perhaps IntegratedHyperParamsAcquisition?

Reply:
Changed to `IntegratedHyperParameterAcquisition`